Polygon_mesh_processing/doc/Polygon_mesh_processing/Concepts/PMPDistanceTraits.h

```diff
@@ -52,6 +55,7 @@ class PMPDistanceTraits{
     /// @name Functions
     /// @{
     Compute_squared_area_3 compute_squared_area_3_object();
+    Compute_squared_area_3 compute_squared_length_3_object();
```

Comment:
Or maybe actually that is a naming issue: should the second one be `Compute_squared_length`?